tests/unit/lib/foxbox/services_test.js

```diff
@@ -0,0 +1,195 @@
+import { waitFor } from '../../test-utils';
```

Comment:
Yes, we need code coverage to know where to focus on writing new unit tests in the future.